openapi/SwarmCommon.yaml

```diff
@@ -1,6 +1,6 @@
 openapi: 3.0.3
 info:
-  version: 3.2.7
+  version: 4.0.0
```

Comment:
curious about why the version bump

Reply:
https://github.com/ethersphere/bee/pull/4627#discussion_r1556346806